data/Music/Sonic the Hedgehog Series/Sonic Chaos/Mecha Green Hill Zone.meta

```diff
@@ -0,0 +1,4 @@
+Sonic Chaos - Mecha Green Hill Zone
+0x03
+bgm
+HyruleField,Fields,WindmillHut
```

Comment:
The location line should have at least one group from each level of specificity (exact, high, mid and low).